api/src/legacy_soap_api/applicants/schemas/opportunity_filter.py

```diff
@@ -0,0 +1,33 @@
+from pydantic import Field, model_validator
+from typing_extensions import Self
```

Comment:
I think this is in `typing` now - don't need to use the extensions library for it.